src/rpc/rpcmapper.rs

```diff
@@ -354,6 +359,7 @@ mod tests {
         Ok(message)
     }
 
+    #[cfg(feature = "cloudevents")]
```

Comment:
I haven't used feature-gating in Rust very much yet. Did you test this locally to make sure these tests are run w/ the feature on and not when the feature is off?